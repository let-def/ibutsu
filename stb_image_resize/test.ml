let string_of_edge = function
  | `CLAMP -> "clamp"
  | `REFLECT -> "reflect"
  | `WRAP -> "wrap"
  | `ZERO -> "zero"

let string_of_filter = function
  | `DEFAULT      -> "default"
  | `BOX          -> "box"
  | `TRIANGLE     -> "triangle"
  | `CUBICBSPLINE -> "cubicbspline"
  | `CATMULLROM   -> "catmullrom"
  | `MITCHELL     -> "mitchell"

let string_of_colorspace = function
  | `LINEAR -> "linear"
  | `SRGB   -> "srgb"

let edge_modes =
  [ `CLAMP; `REFLECT; `WRAP; `ZERO ]

let filters =
  [ `DEFAULT; `BOX; `TRIANGLE; `CUBICBSPLINE; `CATMULLROM; `MITCHELL ]

let colorspaces =
  [ `LINEAR; `SRGB ]

(*let string_of_flag = function
    | `ALPHA_PREMULTIPLIED   -> "alpha_premultiplied"
    | `ALPHA_USES_COLORSPACE -> "alpha_uses_colorspace"
*)

let test ?channels ?alpha_channel name =
  match Stb_image.load ?channels "../test_pokeball.png" with
  | Result.Error (`Msg error) ->
    Printf.printf "KO cannot decode %s: %s\n%!" name error
  | Result.Ok input ->
    let w = Image_type.width input * 3 / 2 in
    let h = Image_type.height input * 3 / 2 in
    let c = Image_type.channels input in
    let buffer =
      Bigarray.Array1.create
        Bigarray.int8_unsigned Bigarray.c_layout (w * h * c)
    in
    let output =
      match Image_type.image ~width:w ~height:h ~channels:c buffer with
      | Result.Ok image -> image
      | Result.Error (`Msg x) -> failwith x
    in
    let with_settings ~edge_mode ~filter ~colorspace () =
      let name = Printf.sprintf "resize-%s-%s-%s-%s.png" name
          (string_of_edge edge_mode) (string_of_filter filter)
          (string_of_colorspace colorspace)
      in
      Stb_image_resize.resize_u8
        ?alpha_channel ~input ~output ~edge_mode ~filter ~colorspace ();
      match Stb_image_write.png name output with
      | Result.Error (`Msg msg) ->
        Printf.printf "KO cannot save %s: %s\n%!" name msg
      | Result.Ok () ->
        let digest = Digest.file name |> Digest.to_hex in
        Printf.printf "OK %s: %s\n%!" name digest
    in
    List.iter (fun edge_mode ->
        List.iter (fun filter ->
            List.iter (fun colorspace ->
                with_settings ~edge_mode ~filter ~colorspace ()
              ) colorspaces
          ) filters
      ) edge_modes

let () = (
  test "";
  test ~channels:1 "1";
  test ~channels:2 "2";
  test ~channels:3 "3";
  test ~channels:4 "4" ~alpha_channel:3;
)
