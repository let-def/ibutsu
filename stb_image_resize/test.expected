OK resize--clamp-default-linear.png: d3138139a4e18825ebd805985bdfe70e
OK resize--clamp-default-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--clamp-box-linear.png: 8b8df18e2f792b2cedfd47e0b3b2be03
OK resize--clamp-box-srgb.png: ebf861e594bba041e8ee7e435f21ba26
OK resize--clamp-triangle-linear.png: 3f17f02c4fabbca2081efe8c9cdb1d0c
OK resize--clamp-triangle-srgb.png: 13bf332eaba226ef42d1e3a539a35021
OK resize--clamp-cubicbspline-linear.png: 1ba3f33718a8a27d1afb62016a1190d2
OK resize--clamp-cubicbspline-srgb.png: 7b7f7ec5bb211d19b72614d6c1c4b550
OK resize--clamp-catmullrom-linear.png: d3138139a4e18825ebd805985bdfe70e
OK resize--clamp-catmullrom-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--clamp-mitchell-linear.png: e6d59199dce86cca34eba7151f395d78
OK resize--clamp-mitchell-srgb.png: 470ca97d17972a30bf2d207d6dc42c42
OK resize--reflect-default-linear.png: f2cc757ff58d6705a27b1ab7ac39fe10
OK resize--reflect-default-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--reflect-box-linear.png: 8b8df18e2f792b2cedfd47e0b3b2be03
OK resize--reflect-box-srgb.png: ebf861e594bba041e8ee7e435f21ba26
OK resize--reflect-triangle-linear.png: fae749159b09d51d2f5207c1433667c6
OK resize--reflect-triangle-srgb.png: e27fa5fca536b7a8898329c4c774620b
OK resize--reflect-cubicbspline-linear.png: e802d3705478cad5c9f23df96da00e83
OK resize--reflect-cubicbspline-srgb.png: bf24395a3eb2b00bc3cdcda190e29f25
OK resize--reflect-catmullrom-linear.png: f2cc757ff58d6705a27b1ab7ac39fe10
OK resize--reflect-catmullrom-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--reflect-mitchell-linear.png: 67fadd73c589c465b08d8a729ea6a72a
OK resize--reflect-mitchell-srgb.png: 1f2fb31b1054f13470a214337ac30844
OK resize--wrap-default-linear.png: 9ed7d21c0e742c7ad6bef7e8c8cfed16
OK resize--wrap-default-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--wrap-box-linear.png: 8b8df18e2f792b2cedfd47e0b3b2be03
OK resize--wrap-box-srgb.png: ebf861e594bba041e8ee7e435f21ba26
OK resize--wrap-triangle-linear.png: e8672fcdcfa250de7ac3a10a99ce5cf1
OK resize--wrap-triangle-srgb.png: a4ee80fec64429cbcf495d0d8db1a11e
OK resize--wrap-cubicbspline-linear.png: 8e6d7a108970bbe0613a3e6e58412330
OK resize--wrap-cubicbspline-srgb.png: 2323a9bd1af9a0f8abf8adcbd0e03cdd
OK resize--wrap-catmullrom-linear.png: 9ed7d21c0e742c7ad6bef7e8c8cfed16
OK resize--wrap-catmullrom-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--wrap-mitchell-linear.png: eaec6fb64941daeb8700387b08db8665
OK resize--wrap-mitchell-srgb.png: 28f8613ee28423f0521645d07af7f4c4
OK resize--zero-default-linear.png: 7bfed537cfce0af616e2c8f2d3cc1a65
OK resize--zero-default-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--zero-box-linear.png: 8b8df18e2f792b2cedfd47e0b3b2be03
OK resize--zero-box-srgb.png: ebf861e594bba041e8ee7e435f21ba26
OK resize--zero-triangle-linear.png: 68a521811816e6cfd608967db651d0e7
OK resize--zero-triangle-srgb.png: 7697083ed29eb147b3bb0ece87744617
OK resize--zero-cubicbspline-linear.png: 41ccfe6c727d3cf6946a43765c2c514a
OK resize--zero-cubicbspline-srgb.png: 0df25488b0746457b30b6dcebdaa35d9
OK resize--zero-catmullrom-linear.png: 7bfed537cfce0af616e2c8f2d3cc1a65
OK resize--zero-catmullrom-srgb.png: de2437d5b87113633bc924b4897e53cf
OK resize--zero-mitchell-linear.png: 1be054e84ff80c69de31bf805253fb36
OK resize--zero-mitchell-srgb.png: a1c5686420cf76381f105559342ee821
OK resize-1-clamp-default-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-clamp-default-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-clamp-box-linear.png: c08a0acabfda1b7919c7718c8d85e55d
OK resize-1-clamp-box-srgb.png: de731170dae6d9cf9428eb90a186f570
OK resize-1-clamp-triangle-linear.png: 61db3ed96ce123ce07d6235c02c13c50
OK resize-1-clamp-triangle-srgb.png: d6f8e7f93b068c0ba090ec857ac8cda6
OK resize-1-clamp-cubicbspline-linear.png: 8bd99a29729d08ebeac61a1d791221cc
OK resize-1-clamp-cubicbspline-srgb.png: 30eae0907bed50af175f9641e237f219
OK resize-1-clamp-catmullrom-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-clamp-catmullrom-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-clamp-mitchell-linear.png: 6a12ba1e7249673e895664c73d2e6ecf
OK resize-1-clamp-mitchell-srgb.png: 6cd96785f0c4d9fd0f4f443fca0fce6d
OK resize-1-reflect-default-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-reflect-default-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-reflect-box-linear.png: c08a0acabfda1b7919c7718c8d85e55d
OK resize-1-reflect-box-srgb.png: de731170dae6d9cf9428eb90a186f570
OK resize-1-reflect-triangle-linear.png: 61db3ed96ce123ce07d6235c02c13c50
OK resize-1-reflect-triangle-srgb.png: d6f8e7f93b068c0ba090ec857ac8cda6
OK resize-1-reflect-cubicbspline-linear.png: 8bd99a29729d08ebeac61a1d791221cc
OK resize-1-reflect-cubicbspline-srgb.png: 30eae0907bed50af175f9641e237f219
OK resize-1-reflect-catmullrom-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-reflect-catmullrom-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-reflect-mitchell-linear.png: 6a12ba1e7249673e895664c73d2e6ecf
OK resize-1-reflect-mitchell-srgb.png: 6cd96785f0c4d9fd0f4f443fca0fce6d
OK resize-1-wrap-default-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-wrap-default-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-wrap-box-linear.png: c08a0acabfda1b7919c7718c8d85e55d
OK resize-1-wrap-box-srgb.png: de731170dae6d9cf9428eb90a186f570
OK resize-1-wrap-triangle-linear.png: 61db3ed96ce123ce07d6235c02c13c50
OK resize-1-wrap-triangle-srgb.png: d6f8e7f93b068c0ba090ec857ac8cda6
OK resize-1-wrap-cubicbspline-linear.png: 8bd99a29729d08ebeac61a1d791221cc
OK resize-1-wrap-cubicbspline-srgb.png: 30eae0907bed50af175f9641e237f219
OK resize-1-wrap-catmullrom-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-wrap-catmullrom-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-wrap-mitchell-linear.png: 6a12ba1e7249673e895664c73d2e6ecf
OK resize-1-wrap-mitchell-srgb.png: 6cd96785f0c4d9fd0f4f443fca0fce6d
OK resize-1-zero-default-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-zero-default-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-zero-box-linear.png: c08a0acabfda1b7919c7718c8d85e55d
OK resize-1-zero-box-srgb.png: de731170dae6d9cf9428eb90a186f570
OK resize-1-zero-triangle-linear.png: 61db3ed96ce123ce07d6235c02c13c50
OK resize-1-zero-triangle-srgb.png: d6f8e7f93b068c0ba090ec857ac8cda6
OK resize-1-zero-cubicbspline-linear.png: 8bd99a29729d08ebeac61a1d791221cc
OK resize-1-zero-cubicbspline-srgb.png: 30eae0907bed50af175f9641e237f219
OK resize-1-zero-catmullrom-linear.png: 87beadcdb865690dd177cc0390e61ff0
OK resize-1-zero-catmullrom-srgb.png: 2f42c91d1549f1f760cf697e518b1733
OK resize-1-zero-mitchell-linear.png: 6a12ba1e7249673e895664c73d2e6ecf
OK resize-1-zero-mitchell-srgb.png: 6cd96785f0c4d9fd0f4f443fca0fce6d
OK resize-2-clamp-default-linear.png: e96de21f39b09f2558a019d21ef2762c
OK resize-2-clamp-default-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-clamp-box-linear.png: 221dc8177237074fffd74e178825a2ab
OK resize-2-clamp-box-srgb.png: 50478e7140a428f68f0d599bb31d60e5
OK resize-2-clamp-triangle-linear.png: 3b23f4837605edcfc7621435321e811c
OK resize-2-clamp-triangle-srgb.png: 8ad2d2570c00ba15e4a53648261d51ef
OK resize-2-clamp-cubicbspline-linear.png: 2eb0424489369e0d89efe8f27d9dc65c
OK resize-2-clamp-cubicbspline-srgb.png: 16d912431c5c23b3bc71980213d73b33
OK resize-2-clamp-catmullrom-linear.png: e96de21f39b09f2558a019d21ef2762c
OK resize-2-clamp-catmullrom-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-clamp-mitchell-linear.png: 5a9a851686e1939cd19cbc306b5f5716
OK resize-2-clamp-mitchell-srgb.png: 0b542d286db9cd18e85126f7cce1d10e
OK resize-2-reflect-default-linear.png: 50b8dcd70f3e28bbe62fdc72326e5f3e
OK resize-2-reflect-default-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-reflect-box-linear.png: 221dc8177237074fffd74e178825a2ab
OK resize-2-reflect-box-srgb.png: 50478e7140a428f68f0d599bb31d60e5
OK resize-2-reflect-triangle-linear.png: bb592a1499f130df90b3834a57cf4c1f
OK resize-2-reflect-triangle-srgb.png: c275c38b3dfa68752ba9170a3b13d24d
OK resize-2-reflect-cubicbspline-linear.png: ad5af76c8c18655d1778370dba32b664
OK resize-2-reflect-cubicbspline-srgb.png: cf9a9bff2fa426d9c2274f867a940676
OK resize-2-reflect-catmullrom-linear.png: 50b8dcd70f3e28bbe62fdc72326e5f3e
OK resize-2-reflect-catmullrom-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-reflect-mitchell-linear.png: a918229b2b7459136929b900f34b7382
OK resize-2-reflect-mitchell-srgb.png: a0fc92371dcbdf300afa65bf173a492f
OK resize-2-wrap-default-linear.png: e27615e861f06a3bc6cf58f1de83b388
OK resize-2-wrap-default-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-wrap-box-linear.png: 221dc8177237074fffd74e178825a2ab
OK resize-2-wrap-box-srgb.png: 50478e7140a428f68f0d599bb31d60e5
OK resize-2-wrap-triangle-linear.png: 76c6124456c053f66f08d6c6ac8240d0
OK resize-2-wrap-triangle-srgb.png: 737ac5c10b33021482cf36645c7dfd71
OK resize-2-wrap-cubicbspline-linear.png: da611017c5c66673eadf6efa764be6d5
OK resize-2-wrap-cubicbspline-srgb.png: 9e7345a0c7c3b747721f2b8cd142a297
OK resize-2-wrap-catmullrom-linear.png: e27615e861f06a3bc6cf58f1de83b388
OK resize-2-wrap-catmullrom-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-wrap-mitchell-linear.png: 09168bd5a233df0883118e386ea5eba3
OK resize-2-wrap-mitchell-srgb.png: 65b8757e65a435d358221deb8c2fe045
OK resize-2-zero-default-linear.png: c9cd69a53d6dad8c344c6c3df2774671
OK resize-2-zero-default-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-zero-box-linear.png: 221dc8177237074fffd74e178825a2ab
OK resize-2-zero-box-srgb.png: 50478e7140a428f68f0d599bb31d60e5
OK resize-2-zero-triangle-linear.png: b15fe9c6db4d19309bf3971e0be18c83
OK resize-2-zero-triangle-srgb.png: 7e1db4e26d031ee640a12aacb24f5749
OK resize-2-zero-cubicbspline-linear.png: 549ee5bf7775feca65d76b64fc157071
OK resize-2-zero-cubicbspline-srgb.png: 379bebc8d2b047e388e4ce2f58447961
OK resize-2-zero-catmullrom-linear.png: c9cd69a53d6dad8c344c6c3df2774671
OK resize-2-zero-catmullrom-srgb.png: a10a91ce6fe96fd0c336d01a3124648a
OK resize-2-zero-mitchell-linear.png: 50cdf833292b40c25aec9a6b25c480a5
OK resize-2-zero-mitchell-srgb.png: 7c0db961b9fa3016bf0852f5f9336cd9
OK resize-3-clamp-default-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-clamp-default-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-clamp-box-linear.png: e059dd8dbd9dd9bbecb5de1f188f50ef
OK resize-3-clamp-box-srgb.png: 80f13123f48e01bb44c30d5fbbf19f62
OK resize-3-clamp-triangle-linear.png: eef58245c1ea7c5bb647feb04efaeb8c
OK resize-3-clamp-triangle-srgb.png: 82755613fd6cfb12b1ce4e13cd48821c
OK resize-3-clamp-cubicbspline-linear.png: 5e6c4eb4b5138f4070ae41cace6d8760
OK resize-3-clamp-cubicbspline-srgb.png: 4ef2579010903a9dee2a0c25423e6af0
OK resize-3-clamp-catmullrom-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-clamp-catmullrom-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-clamp-mitchell-linear.png: 9191e9e69a397f45c7e2b6106f558cbe
OK resize-3-clamp-mitchell-srgb.png: 4cbcda1260c520a27e5d42137bd4b34a
OK resize-3-reflect-default-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-reflect-default-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-reflect-box-linear.png: e059dd8dbd9dd9bbecb5de1f188f50ef
OK resize-3-reflect-box-srgb.png: 80f13123f48e01bb44c30d5fbbf19f62
OK resize-3-reflect-triangle-linear.png: eef58245c1ea7c5bb647feb04efaeb8c
OK resize-3-reflect-triangle-srgb.png: 82755613fd6cfb12b1ce4e13cd48821c
OK resize-3-reflect-cubicbspline-linear.png: 5e6c4eb4b5138f4070ae41cace6d8760
OK resize-3-reflect-cubicbspline-srgb.png: 4ef2579010903a9dee2a0c25423e6af0
OK resize-3-reflect-catmullrom-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-reflect-catmullrom-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-reflect-mitchell-linear.png: 9191e9e69a397f45c7e2b6106f558cbe
OK resize-3-reflect-mitchell-srgb.png: 4cbcda1260c520a27e5d42137bd4b34a
OK resize-3-wrap-default-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-wrap-default-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-wrap-box-linear.png: e059dd8dbd9dd9bbecb5de1f188f50ef
OK resize-3-wrap-box-srgb.png: 80f13123f48e01bb44c30d5fbbf19f62
OK resize-3-wrap-triangle-linear.png: eef58245c1ea7c5bb647feb04efaeb8c
OK resize-3-wrap-triangle-srgb.png: 82755613fd6cfb12b1ce4e13cd48821c
OK resize-3-wrap-cubicbspline-linear.png: 5e6c4eb4b5138f4070ae41cace6d8760
OK resize-3-wrap-cubicbspline-srgb.png: 4ef2579010903a9dee2a0c25423e6af0
OK resize-3-wrap-catmullrom-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-wrap-catmullrom-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-wrap-mitchell-linear.png: 9191e9e69a397f45c7e2b6106f558cbe
OK resize-3-wrap-mitchell-srgb.png: 4cbcda1260c520a27e5d42137bd4b34a
OK resize-3-zero-default-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-zero-default-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-zero-box-linear.png: e059dd8dbd9dd9bbecb5de1f188f50ef
OK resize-3-zero-box-srgb.png: 80f13123f48e01bb44c30d5fbbf19f62
OK resize-3-zero-triangle-linear.png: eef58245c1ea7c5bb647feb04efaeb8c
OK resize-3-zero-triangle-srgb.png: 82755613fd6cfb12b1ce4e13cd48821c
OK resize-3-zero-cubicbspline-linear.png: 5e6c4eb4b5138f4070ae41cace6d8760
OK resize-3-zero-cubicbspline-srgb.png: 4ef2579010903a9dee2a0c25423e6af0
OK resize-3-zero-catmullrom-linear.png: 4cf3f1bc63dade8147ab3659bad571f2
OK resize-3-zero-catmullrom-srgb.png: 37299f647c10ab0f847311b4a954b999
OK resize-3-zero-mitchell-linear.png: 9191e9e69a397f45c7e2b6106f558cbe
OK resize-3-zero-mitchell-srgb.png: 4cbcda1260c520a27e5d42137bd4b34a
OK resize-4-clamp-default-linear.png: 20a7164bd12158732a10622625af72e3
OK resize-4-clamp-default-srgb.png: ae507b2bb7d27eefc6a509bc2e23f42a
OK resize-4-clamp-box-linear.png: 42c2c69bde699219d0746a2d95f75ce5
OK resize-4-clamp-box-srgb.png: 17ccdd1c14b3b57ca2b71133e72d82e0
OK resize-4-clamp-triangle-linear.png: a8041d8e2707eb4093ab959674c561cc
OK resize-4-clamp-triangle-srgb.png: ccfd125ef6e26ac9147fa0e54ee2e993
OK resize-4-clamp-cubicbspline-linear.png: 455fd427dd17b3bda96578bc4e8d616a
OK resize-4-clamp-cubicbspline-srgb.png: 97a3fefc74b7ab8ef2a3d599614b0f4d
OK resize-4-clamp-catmullrom-linear.png: 20a7164bd12158732a10622625af72e3
OK resize-4-clamp-catmullrom-srgb.png: ae507b2bb7d27eefc6a509bc2e23f42a
OK resize-4-clamp-mitchell-linear.png: 1eb28e0abf4d47c212cbba3520605025
OK resize-4-clamp-mitchell-srgb.png: 84594b3a7acefe7163a476bf77ee647d
OK resize-4-reflect-default-linear.png: b9ba3f10df0be4078f7f421137847079
OK resize-4-reflect-default-srgb.png: a97f57b256dfb4c55847f9b6d0133a6f
OK resize-4-reflect-box-linear.png: 42c2c69bde699219d0746a2d95f75ce5
OK resize-4-reflect-box-srgb.png: 17ccdd1c14b3b57ca2b71133e72d82e0
OK resize-4-reflect-triangle-linear.png: 8020ed463009328e93e341693895d9fe
OK resize-4-reflect-triangle-srgb.png: fa803504271e4476de85b5c79c905eb9
OK resize-4-reflect-cubicbspline-linear.png: 428ba20207c15cf51e9e845fb8afe08b
OK resize-4-reflect-cubicbspline-srgb.png: 20253de08d36ebae943f4260193a6467
OK resize-4-reflect-catmullrom-linear.png: b9ba3f10df0be4078f7f421137847079
OK resize-4-reflect-catmullrom-srgb.png: a97f57b256dfb4c55847f9b6d0133a6f
OK resize-4-reflect-mitchell-linear.png: 9c78855940a94a57b9c62a4f453285fa
OK resize-4-reflect-mitchell-srgb.png: a81cce0cc7b6234a4109159d0f32febc
OK resize-4-wrap-default-linear.png: 1ff78cfa37ef8d5a5810d530c2d93d3d
OK resize-4-wrap-default-srgb.png: 4544c6eba5416c57312f3085223b0d42
OK resize-4-wrap-box-linear.png: 42c2c69bde699219d0746a2d95f75ce5
OK resize-4-wrap-box-srgb.png: 17ccdd1c14b3b57ca2b71133e72d82e0
OK resize-4-wrap-triangle-linear.png: cbc7ce99a7aebf1aadf550423b81c553
OK resize-4-wrap-triangle-srgb.png: 87c40567559c4ba81c457d669cd74512
OK resize-4-wrap-cubicbspline-linear.png: ff280008caf86b4af3058aa807c3fc6b
OK resize-4-wrap-cubicbspline-srgb.png: dc32959ab01dfbda4e20affa99415a58
OK resize-4-wrap-catmullrom-linear.png: 1ff78cfa37ef8d5a5810d530c2d93d3d
OK resize-4-wrap-catmullrom-srgb.png: 4544c6eba5416c57312f3085223b0d42
OK resize-4-wrap-mitchell-linear.png: 546f907893ce8b98d3b14d60673017dd
OK resize-4-wrap-mitchell-srgb.png: 37e19641bb8a7959f5f384e325e32aa8
OK resize-4-zero-default-linear.png: f453514c113ea4f986218aab6e446574
OK resize-4-zero-default-srgb.png: eb4c3f78ce7b5d064b7d3700458b1393
OK resize-4-zero-box-linear.png: 42c2c69bde699219d0746a2d95f75ce5
OK resize-4-zero-box-srgb.png: 17ccdd1c14b3b57ca2b71133e72d82e0
OK resize-4-zero-triangle-linear.png: 768b15ade376ff1d32f09125fd506a7c
OK resize-4-zero-triangle-srgb.png: f6b7c3ce7ed625a2ce14f82ff86cbf44
OK resize-4-zero-cubicbspline-linear.png: 574a081c2636602ced974b6661c31149
OK resize-4-zero-cubicbspline-srgb.png: 7bf5145e862303b679029b73e1816b20
OK resize-4-zero-catmullrom-linear.png: f453514c113ea4f986218aab6e446574
OK resize-4-zero-catmullrom-srgb.png: eb4c3f78ce7b5d064b7d3700458b1393
OK resize-4-zero-mitchell-linear.png: 7b691d95ca751b9ecd6f2bca5db43640
OK resize-4-zero-mitchell-srgb.png: b1798672d1ce26d0a93c051e26c1d12c
